lambdas/sqs-message-remover/tests/test-index.js

```diff
@@ -1,17 +1,89 @@
 'use strict';
 
 const delay = require('delay');
-const rewire = require('rewire');
 const test = require('ava');
 const get = require('lodash/get');
 
 const awsServices = require('@cumulus/aws-client/services');
 const { receiveSQSMessages } = require('@cumulus/aws-client/SQS');
 const { randomString } = require('@cumulus/common/test-utils');
-const { createSqsQueues, getSqsQueueMessageCounts } = require('../../lib/testUtils');
 
-const sqsMessageRemover = rewire('../../lambdas/sqs-message-remover');
-const updateSqsQueue = sqsMessageRemover.__get__('updateSqsQueue');
+const { updateSqsQueue } = require('..');
+
+// TODO: Copied from @cumulus/api/lib/testUtils to avoid the dependency, but
```

Comment:
Is this still going to be done?

Reply:
I figured this could be done in another PR after this one. In the spirit of not creating huge PRs, etc. I could add it to this PR

It seems like we should have a `packages/test-utils` package with the stuff from this file and `common/test-utils`.

Reply:
Alright, I'm fine with that as long as it doesn't get forgotten.